src/queue.ts

```diff
@@ -82,13 +83,13 @@ export class AmqpQueue<T> extends EventEmitter {
    * @returns {Promise<{ content: any; properties: PublishOptions }>}
    * @memberof AmqpQueue
    */
+  // tslint:disable-next-line:max-line-length
```

Comment:
I don't think this is needed, this line is just shy of the 120 mark

Reply:
Woops, missed that one. The line was longer when I did first pass. Removed it now 👍